lua/neogit/operations.lua

```diff
@@ -0,0 +1,30 @@
+local a = require('neogit.async')
+-- This is a table to look up pending neogit operations.
```

Comment:
What is your definition of an "neogit operation"?

Reply:
that's a very loose notion there, i basically just mean "top-level operations", those that are directly invoked by the end user. Think "commit", "switch branch", "stash", etc. As mentioned it's really just a stop-gap implementation, since I need a way to wait for those operations from the test execution.

Reply:
Could you please add this to the comment? I will probably forget what this was about in 1 month or so 😆